<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventView</class>
 <widget class="QDialog" name="EventView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="DateLabel">
          <property name="text">
           <string>Data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="AllDayCheckBox">
        <property name="text">
         <string>Tutto Il giorno</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="StartTimeLabel">
          <property name="text">
           <string>Orario di inzio</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="StartTimeEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="EndTimeLabel">
          <property name="text">
           <string>Orario di fine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="EndTimeEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="NoteLabel">
      <property name="text">
       <string>Note</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="NoteEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DeleteButton">
      <property name="text">
       <string>Cancella Evento</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
